build/latest/arm-12.1.0.config

```diff
@@ -645,6 +695,12 @@ CT_ALL_CC_CHOICES="GCC"
 #
 CT_CC_LANG_CXX=y
 CT_CC_LANG_FORTRAN=y
+# CT_CC_LANG_ADA is not set
```

Comment:
Is this correct?

Reply:
Yes. For arm target, I'm using Alire's compilers. But good point, as I think Alire only provides 32-bit target, not aarch64 (and I've also disabled Ada on aarch64).